src/main/java/org/casbin/casdoor/config/CasdoorConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package org.casbin.casdoor.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author tangyang9464
+ */
+@ConfigurationProperties(prefix = "casdoor")
+@Component
```

Comment:
Configuration classes specified with the @EnableConfigurationProperties annotation are automatically injected without the @Component annotation.